install.yml

```
@@ -336,8 +336,38 @@ onAfterClone:
       res = jelastic.dev.scripting.eval("appstore", session, "InstallApp", data);
       return {"result": 0, "out": res}
 
+onBeforeDelete:
+  deleteEnvLink
+
 success: /text/success.md
 
+# -- Actions --
+actions:
+  deleteEnvLink:
+    - script: |-
+        const envName = "${env.envName}";
+        jCustomerEnv = jelastic.env.control.GetNodeGroups(envName, session).object.filter(function (object) {
+                                return object.name == "cp";
+                              }).pop().envLink;
+        if (jCustomerEnv == null) {
+          return {"result": 0, "out": "No jCustomer env linked"};
+        }
+        envsLinked = jelastic.env.control.GetNodeGroups(jCustomerEnv, session).object.filter(function (object) {
+                                return object.name == "cp";
+                              }).pop().envLink;
+        if (envsLinked.indexOf(envName) == -1) {
+          return {"result": 0, "out": envName + " not in envLink of " + jCustomerEnv};
+        }
+        if (envsLinked.indexOf(",") != -1) {      // envLink can contain multiple Jahia envs on jCustomer side
+          envsLinkedArr = envsLinked.split(",");
+          removedEnv = envsLinkedArr.splice(envsLinkedArr.indexOf(envName), 1);
```

`removedEnv` is never used, it can be removed:
```suggestion
          envsLinkedArr.splice(envsLinkedArr.indexOf(envName), 1);
```